schemas/context/media-timed.schema.json

```diff
@@ -31,84 +31,133 @@
           "description": "Timed media chapter information."
         },
         "xdm:impressions": {
-          "title": "Impressions",
-          "$ref": "https://ns.adobe.com/xdm/data/measure",
```

Comment:
Is this a breaking change to remove `$ref`? If so I have concerns about this change.

Reply:
I have confirmed this is a breaking change. I would prefer that we only add to the schema rather than break backwards compatibility.

Reply:
Reverted the changes for removing the references to measure so we can separate the two issues we were working on. This PR now only contains the addition of new fields.

Reply:
Thank you for making this additive only! I have no further concerns.